Alexey/Text/check_if_palindrome/main/check_if_palindrome.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+
+def main():
+    sentence = int(input("Please input your sentence: "))
+    print(check_if_palindrome(sentence))
+
+
+def check_if_palindrome(sentence):
+    sentence = re.sub(r'[^\w]', '', sentence).lower()
+    if sentence == reverse(sentence):
+        massage = "Congrats your sentence is palindrome!"
+        return massage
+    else:
+        massage = "This is not palindrome=("
+        return massage
+
+
+def reverse(sentence):
```

Comment:
there is no need to create one-line function here. 
`if sentence == reverse(sentence):`
Can be `if sentence == sentence[::-1]`
Or look even better 
```
reversed_value = sentence[::-1]
if sentence == reversed_value
```